adafruit_led_animation/__init__.py

```diff
@@ -7,36 +7,3 @@
 
 Author(s): Kattni Rembor
 """
-
-try:
-    from micropython import const
-except ImportError:
-
-    def const(value):  # pylint: disable=missing-docstring
-        return value
```

Comment:
@jepler You did not include this in your updated version. Is it no longer necessary?